### PR TITLE
changed database Sqlite3 to Postgresql

### PR DESCRIPTION
**Pull request by mohseengs**
**Changelog :**
1. changed default Sqlite3 db to postgresql

### CODE REVIEW

File: Musico/config/database.yml [hunk 1]
```diff
@@ -1,25 +1,92 @@
...
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: Musico_development
+  username: mohseen
+  password: mohseentech
```
Comment (mgandhi-gammastack) [inline Musico/config/database.yml]: use env to read username and password, and keep default values as `postgres`

Comment (mohseengs) [reply Musico/config/database.yml]: done

File: Musico/config/database.yml [hunk 2]
```diff
@@ -1,25 +1,92 @@
...
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: Musico_development
+  username: <%= ENV["DB_USERNAME"] %>
+  password: <%= ENV["DB_PASSWORD"] %>
```
Comment (mgandhi-gammastack) [inline Musico/config/database.yml]:
```suggestion
  username: <%= ENV["DB_USERNAME"] || 'postgres' %>
  password: <%= ENV["DB_PASSWORD"] || 'postgres' %>
```

```suggestion
  username: <%= ENV.fetch("DB_USERNAME", 'postgres') %>
  password: <%= ENV.fetch("DB_PASSWORD", 'postgres') %>
```

**Review by mgandhi-gammastack — APPROVED** [review]
left one comment

**Pull request merged**